README.md

```diff
@@ -127,7 +127,8 @@ deployment that are specific to your environment.
 | `keypair_path`      | :heavy_check_mark: |         | The path to the private key for your AWS key pair.<br><br>Note: The private key's integrity will be maintained. It is only used by Ansible to connect to the bastion host.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | `openshift_version` | :heavy_check_mark: |         | The OpenShift version to install.<br><br>The version must include the z-stream (e.g. 4.3.18)  The list of versions can be found [here][openshift_versions]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
-| `pull_secret`       | :heavy_check_mark: |         | The content of your pull secret, which can be found [here][pull_secret]. Be sure to wrap its value in single quotes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `pull_secret`       | :heavy_check_mark: |         | The content of your pull secret, which can be found [here][pull_secret]. Be sure to wrap its value in single quotes.                                                                                                                                                                                                                                                                                                                                  |
+| `install_fips_mode`       | :heavy_check_mark: |         | Set to True to install a cluster that has FIPS-validation enabled                                                                                                                                                                                       |
```

Comment:
Align all of the vertical bars for the table.

For example:

```markdown
| `pull_secret`       | :heavy_check_mark: |         | content      |
| `install_fips_mode` | :heavy_check_mark: |         | more content |
| `rhcos_ami`         | :heavy_check_mark: |         | blah         |
```
If you have text wrapping turned on in your editor, turn it off to help view things for the table.